themes/docsy/layouts/partials/stale-notice.html

```diff
@@ -0,0 +1,3 @@
+{{ if now.After (.context.Params.lastmod.AddDate 1 0 0) }}
+    <div class="alert alert-warning">This {{ .contentType }} is over a year old and may not be completely up-to-date.</div>
```

Comment:
Not sure I like the use of "completely" here. What's the difference between potentially:
- not being up to date, or
- not being completely up to date

Reply:
Ye not tied to either way. Was just trying to 'soften' the language / impact so people don't just go 'well this is useless then!'